db/models.py

```diff
@@ -113,13 +114,24 @@ class Meta():
 
     monitor_user_id = fields.IntField(pk=True)
     user_id = fields.BigIntField()
-    
+
+class StaffMonitorMessageGroups(Model):
+    class Meta():
+        table = "monitor_message_groups"
+
+    group_id = fields.IntField(pk=True)
+    name = fields.CharField(max_length=256)
+    disabled = fields.BooleanField(default=False)
+    monitor_messages: fields.ManyToManyRelation["StaffMonitorMessage"] = fields.ManyToManyField('models.StaffMonitorMessage', related_name="groups")
+
 class StaffMonitorMessage(Model):
     class Meta():
         table = "monitor_message"
 
     monitor_message_id = fields.IntField(pk=True)
+    disabled = fields.BooleanField(default=False)
     message = fields.CharField(max_length=1000)
+    groups: fields.ManyToManyRelation[StaffMonitorMessageGroups]    
```

Comment:
Is this field used?

Reply:
Not currently used no. I had a feeling it was an internal field used by TortoiseORM but it's more used for reverse relation queries (i.e., checking which groups monitor message patterns are under). Can be useful but we're not using it at the moment - is it better that we remove it?

Reply:
Hmmm, does it currently get populated when you add a message to a group? Otherwise I feel like its not worth keeping.

Reply:
There's no field on the table itself for [`groups`](https://user-images.githubusercontent.com/18462634/214722545-f49cfa2c-4593-4bff-a735-ef66b032cbb3.png) but there is a secondary table made that accounts for these relations seen below.

![image](https://user-images.githubusercontent.com/18462634/214722769-5a729a2e-6790-406d-898e-2c556f42d3a8.png)

Reply:
Hmm, so we'd be able to add it back if we did need it right? If so, then I reckon we remove it.

Reply:
Yeah I'd think so - can remove in next commit. Though I can't make a migration for some reason, so I'm unsure how this actually affects the database.

Reply:
If its too much hassle, don't worry about it I reckon.

Reply:
Turns out it's a bug with [`aerich`](https://github.com/tortoise/aerich/issues/150). After running the migration, nothing was changed for the DB so my hunch about the internal field was correct. The secondary table is made from the other relation statement. I'll remove it in the next commit.